src/BlazorSignalR/BlazorSignalR.csproj

```diff
@@ -8,14 +8,14 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);IncludeP2POutput</TargetsForTfmSpecificBuildOutput>
-    <Version>0.9.0-blazor-3.0.0-preview8.19405.7</Version>
+    <Version>0.10.0-blazor-3.0.0-3.0.0-preview9.19424.4</Version>
```

Comment:
Oh! This is my mistake.
It should be "0.10.0-blazor-3.0.0-preview9.19424.4", not "0.10.0-blazor-3.0.0-3.0.0-preview9.19424.4".